protocol/0044-LIME-lp_mechanics.md

```diff
@@ -155,7 +155,7 @@ $$
 Transfer $\text{maximum-penalty-free-reduction-amount}_i$ to their general account.
 
 Now transfer $(1-\text{market.liquidity.earlyExitPenalty}) \cdot \text{penalty-incurring-reduction-amount}_i$ to their general account and transfer $\text{market.liquidity.earlyExitPenalty} \cdot  \text{penalty-incurring-reduction-amount}_i$ to the market insurance pool.
-Note that in the case of spot market or any market that isn't running an insurance pool the transfer should go into the network treasury for the asset.
+Note that in the case of spot market or any market that isn't running an insurance pool the transfer should go into the network treasury for the asset [TODO (WG): this is indeed how it's implemented now, but do want to leave it like that or transfer funds into the global insurance pool instead?].
```

Comment:
This is indeed how it's implemented now, but do want to leave it like that or transfer funds into the global insurance pool instead?

Reply:
Sounds like a reasonable future change, though not one I foresee much appetite to spend time on this year.